team-config.yml

```diff
@@ -269,3 +269,9 @@ camunda:
   slack:
     contact_channel: "#rpe"
     build_notices_channel: "#rpe-build-notices"
+ethos:
+  team: "Ethos Replacement"
+  namespace: "ethos"
+  slack:
+    contact_channel: "#ethos-replacement"
```

Comment:
I can't find this channel?
A contact channel should be public

Reply:
Ok, sorry, updated with a public channel